subcommands/common.go

```diff
@@ -132,6 +132,30 @@ func Tabby(indent int, columns ...interface{}) *tabby.Tabby {
 	return tab
 }
 
+func AssertWritable(path string) {
+	st, err := os.Stat(path)
+	DieNotNil(err)
+	f, err := os.OpenFile(path, os.O_APPEND|os.O_WRONLY, st.Mode())
+	if err != nil {
+		fmt.Println("ERROR: File is not writeable:", path)
+		os.Exit(1)
+	}
+	f.Close()
+}
+
+func IsSliceSetEqual(first, second []string) bool {
```

Comment:
Will it work if the slices have different number of the same value, e.g. []{'foo', 'foo'} and []{'foo'}? I guess we assume that elements are unique in the slices.

Reply:
Also, it won't work if the first slice in longer, e.g. []{'foo', 'bar'} and []{'foo'}, the function considers them equal.

Reply:
oh, this is just code reshuffling so I suppose it works for our use-cases.

Reply:
I was just reshuffling; but based on your comments I have to look for bugs now.

Reply:
Actually, it was a good catch; thanks @mike-sul.
https://github.com/foundriesio/fioctl/pull/184/commits/de529e85e568c9fb268cc3ba45dba6d1950ee5f7

Reply:
It should work if elements in the slices are unique, otherwise `int`/counter should be used instead of boolean.

Reply:
This is not `IsSliceEqual`, it is `IsSliceSetEqual` i.e. it compares sets, not samples; and in a set logic `set("foo, foo") == set("foo")`.

That is exactly what we want from a function named that way.